{{cookiecutter.github_repository}}/setup.cfg

```diff
@@ -12,7 +12,9 @@ tag = True
 [bumpversion:file:README.md]
 
 [flake8]
-max-line-length = 120
+ignore = E203, E266, E501, W503
```

Comment:
Why are we ignoring these? 

E203 has whitespace before `:` which is what the recommendation says.

Reply:
https://github.com/psf/black/issues/315
Maybe because of this ?

Reply:
Akash yes it's because of that, more official doc: https://github.com/psf/black#slices

Reply:
I see. Looks good.